key-manager/src/main/scala/org/bitcoins/keymanager/KeyManagerCreateApi.scala

```diff
@@ -16,22 +16,21 @@ import scodec.bits.BitVector
   *                           can write it down. They should also be prompted
   *                           to confirm at least parts of the code.
   */
-trait KeyManagerCreateApi {
+trait KeyManagerCreateApi[T <: KeyManager] {
```

Comment:
nit: `KM` at the very least instead of `T` if not something even more descriptive